includes/class-amp-travel-cpt.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Class for Travel Custom Post Type.
+ *
+ * @package WPAMPTheme
+ */
+
+/**
+ * Class AMP_Travel_CPT
+ *
+ * @package WPAMPTheme
+ */
+class AMP_Travel_CPT {
+
+	/**
+	 * The post type single slug.
+	 *
+	 * @var string
+	 */
+	const POST_TYPE_SLUG_SINGLE = 'adventure';
+
+	/**
+	 * The post type plural slug.
+	 *
+	 * @var string
+	 */
+	const POST_TYPE_SLUG_PLURAL = 'adventures';
+
+	/**
+	 * Initialize the CPT class.
+	 */
+	public function init() {
+		add_action( 'init', array( $this, 'setup' ) );
+	}
+
+	/**
+	 * Setup the Custom post type support.
+	 */
+	public function setup() {
+		/**
+		 * Enable support for Post Thumbnails on posts and pages.
+		 *
+		 * @link https://developer.wordpress.org/themes/functionality/featured-images-post-thumbnails/
+		 */
+		add_theme_support( 'post-thumbnails' );
+
+		/**
+		 * Filter the image sizes to allow manipulating and adding sizes to be registered.
+		 *
+		 * @param array $sizes The array of sizes to be registered.
+		 */
+		$image_sizes = apply_filters( 'amp_travel_image_sizes', array(
+			'1600x900',
+			'1400x787',
+			'1200x675',
+			'1040x585',
+			'768x432',
+			'727x409',
+			'600x338',
+			'500x281',
+			'375x211',
+			'335x188',
+			'320x180',
+			'280x158',
+			'240x135',
+			'160x90',
+			'122x67',
```

Comment:
Are all of these image sizes needed? 

You know more about the specifics than I do. But it seems like a lot to create 15 more images sizes for every image uploaded.

Reply:
@kienstra - no idea. but it's a good point. I took these from the `ampconf` theme as a starting point. I'll reduce it down to what I think should be needed for the current blocks, etc.

Reply:
This is something that we should revisit. But I'm going to merge this PR to unblock other PRs.